health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -777,13 +919,391 @@
+    },
+    additionalCustomizations: (row, key, column, value, t, searchResult) => {
+      const [showPopup, setShowPopup] = useState(false);
+      const FacilityPopUp = Digit.ComponentRegistryService.getComponent("FacilityPopup");
+
+      switch (key) {
+        case "MICROPLAN_FACILITY_SERVINGPOPULATION":
+          return row?.additionalDetails?.servingPopulation;
+        case "MICROPLAN_FACILITY_RESIDINGVILLAGE":
+          return t(row?.residingBoundary);
+        case "MICROPLAN_FACILITY_ASSIGNED_VILLAGES":
+          const assignedVillages = row?.serviceBoundaries;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)